apps/mitt-konto/index.js

```diff
@@ -2,15 +2,10 @@ import React from "react";
 import ReactDOM from "react-dom";
 import "basscss/css/basscss-cp.css";
 
-// yup, welcome to react 16
-import createReactClass from "create-react-class";
-React.createClass = createReactClass;
```

Comment:
A handful of services define `createClass` in their index.js files. None of the generated code refers to that function.